apitesters/offerings.ts

```diff
@@ -56,7 +56,7 @@ function checkPackage(pack: PurchasesPackage) {
 function checkOffering(offering: PurchasesOffering) {
   const identifier: string = offering.identifier;
   const serverDescription: string = offering.serverDescription;
-  const metadata: Map<string, any> = offering.metadata;
+  const metadata: { [key: string]: unknown } = offering.metadata;
```

Comment:
I've been reading a bit about any vs unknown. Unknown forces developers to have to check the type before doing anything with the value, whereas `any` basically disables type checks and devs can do whatever they want. I think `unknown` is more appropriate but lmk if you think otherwise.